### DISCUSSION TITLE
How to get an SOH vs time curve for Li ion battery in PyBAMm

### DISCUSSION THREAD

**Discussion by alihassan90** [pybamm-team/PyBaMM#2963] [General]
I want to perform ALT on a Lithium ion battery by giving it a charge discharge pattern over a certain period of time (year) and see how capacity fades over the time. Can it be done in PyBAMm. 

Thanks

**Comment by brosaplanella** [comment]
Yes, [this notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/simulating-long-experiments.ipynb) provides a good starting point for this.
